try {
  assert(1 + 1 == 3);
} catch (any exception) {
  print(exception);
  for (var kv : exception) {
    print(kv);
  }
}
///><RESOURCE>(2,10-19): Assertion is untrue: 2 == 3
///>{key:"message",value:"Assertion is untrue: 2 == 3"}
///>{key:"resource",value:"<RESOURCE>"}
///>{key:"line",value:2}
///>{key:"column",value:10}
///>{key:"left",value:2}
///>{key:"operator",value:"=="}
///>{key:"right",value:3}
